src/webapi/window.rs

```diff
@@ -3,6 +3,26 @@ use webapi::event_target::{IEventTarget, EventTarget};
 use webapi::window_or_worker::IWindowOrWorker;
 use webapi::storage::Storage;
 use webapi::location::Location;
+use webcore::try_from::TryInto;
+use webcore::serialization::Once;
+use webcore::value::Value;
+
+/// A handle to a pending animation frame request.
+#[derive(Debug)]
+pub struct RequestAnimationFrameHandle(Value);
+
+impl RequestAnimationFrameHandle {
+    /// Cancels an animation frame request.
+    ///
+    /// [(Javascript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Window/cancelAnimationFrame)
+    pub fn cancel(self) {
+        js!{
+            let val = @{self.0};
+            cancelAnimationFrame(val.request);
```

Comment:
This should probably also be called on the same `Window` instance which was used in `request_animation_frame` instead of on the globally implicit `window`. (Again, I don't know if this *really* matters or will ever matter in practice; we can always change it later. Since you're passing a `Value` around putting a reference to the `window` in there is going to be essentially free anyway.)